### PR TITLE
Adding revoke rotated refresh token option

### PR DESCRIPTION
**Pull request by basz**
part of #54

### CODE REVIEW

File: src/Container/RefreshTokenGrantFactory.php [hunk 1]
```diff
@@ -35,12 +36,19 @@ class RefreshTokenGrantFactory
...
         /* @var AccessTokenService $accessTokenService */
         $accessTokenService = $container->get(AccessTokenService::class);
 
         /* @var RefreshTokenService $refreshTokenService */
         $refreshTokenService = $container->get(RefreshTokenService::class);
 
-        return new RefreshTokenGrant($accessTokenService, $refreshTokenService);
+        $service = new RefreshTokenGrant($accessTokenService, $refreshTokenService);
+
+        $service->setRevokeRotatedRefreshToken($serverOptions->getRevokeRotatedRefreshTokens());
```
Comment (ojhaujjwal) [inline src/Container/RefreshTokenGrantFactory.php]: Why not just pass that option as a constructor argument?

**Comment by basz** [comment]
I did the same thing as other factories do, but would be happy to change it.

Except the next obvious question would be; why not pass the ServerOption instance in the constructor? Ill propose to make a new separate issue for that...

**Comment by ojhaujjwal** [comment]
@basz  good point. go for it.

File: src/Container/RefreshTokenGrantFactory.php (hunk 1 above)
Comment (bakura10) [inline src/Container/RefreshTokenGrantFactory.php]: +1

**Comment by bakura10** [comment]
I agree the option should be injected. However for now the feature looks good so I'm merging it !

**Pull request merged**